UI/MapWnd.cpp

```diff
@@ -5779,16 +5779,36 @@ void MapWnd::RefreshFleetResourceIndicator() {
     const std::set<int>& this_client_known_destroyed_objects = GetUniverse().EmpireKnownDestroyedObjectIDs(empire_id);
 
     int total_fleet_count = 0;
+    int total_armed_ships = 0;
+    int total_ship_slot_count = 0;
+    int total_ship_part_count = 0;
     for (TemporaryPtr<const Ship> ship : Objects().FindObjects<Ship>()) {
-        if (ship->OwnedBy(empire_id) && this_client_known_destroyed_objects.find(ship->ID()) == this_client_known_destroyed_objects.end())
+        if (ship->OwnedBy(empire_id) &&
+            this_client_known_destroyed_objects.find(ship->ID()) == this_client_known_destroyed_objects.end())
+        {
             total_fleet_count++;
+            if (ship->IsArmed())
+                total_armed_ships++;
+            const ShipDesign* design = ship->Design();
+            if (!design)
+                continue;
+            total_ship_part_count += design->Parts().size();
+            const HullType* hull_type = design->GetHull();
+            if (!hull_type)
+                continue;
+            total_ship_slot_count += hull_type->NumSlots();
+        }
     }
 
     m_fleet->SetValue(total_fleet_count);
     m_fleet->ClearBrowseInfoWnd();
     m_fleet->SetBrowseModeTime(GetOptionsDB().Get<int>("UI.tooltip-delay"));
     m_fleet->SetBrowseInfoWnd(boost::shared_ptr<GG::BrowseInfoWnd>(
-        new TextBrowseWnd(UserString("MAP_FLEET_TITLE"), UserString("MAP_FLEET_TEXT"))));
+        new TextBrowseWnd(UserString("MAP_FLEET_TITLE"),
```

Comment:
Done. PR #1210 (will drop for this PR when updated with other changes requested)